packages/bierzo-wallet/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bierzo-wallet",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^3.9.3",
+    "@types/jest": "24.0.11",
+    "@types/node": "11.12.2",
+    "@types/react": "16.8.10",
+    "@types/react-dom": "16.8.3",
+    "medulas-react-components": "^0.0.0",
+    "normalize.css": "^8.0.1",
+    "react-scripts": "2.1.8",
+    "react": "^16.8.6",
```

Comment:
💥 General libraries should be moved to root's lerna monorepo. At least the extension is using react, so please move up.

Reply:
Fixed by [14f6201](https://github.com/iov-one/ponferrada/pull/79/commits/14f6201854bd623d3fb74608509f7e5b7c587dc8)